spec/.packageWithStart.json

```diff
@@ -0,0 +1,62 @@
+{
```

Comment:
Any reason this is a dotfile?

Reply:
We can make them normal files :) No idea why they're dotfiles!